instrumentation/openai/openai-java-1.1/testing/src/main/java/io/opentelemetry/instrumentation/openai/v1_1/AbstractEmbeddingsTest.java

```diff
@@ -0,0 +1,252 @@
+            metric ->
+                metric
+                    .hasName("gen_ai.client.operation.duration")
+                    .hasHistogramSatisfying(
+                        histogram ->
+                            histogram.hasPointsSatisfying(
+                                point ->
+                                    point
+                                        .hasSumGreaterThan(0.0)
+                                        .hasAttributesSatisfyingExactly(
+                                            equalTo(GEN_AI_SYSTEM, OPENAI),
+                                            equalTo(GEN_AI_OPERATION_NAME, EMBEDDINGS),
+                                            equalTo(GEN_AI_REQUEST_MODEL, MODEL),
+                                            equalTo(GEN_AI_RESPONSE_MODEL, MODEL)))),
+            metric ->
+                metric
+                    .hasName("gen_ai.client.token.usage")
+                    .hasHistogramSatisfying(
+                        histogram ->
+                            histogram.hasPointsSatisfying(
+                                point ->
+                                    point
+                                        .hasSum(4.0)
+                                        .hasAttributesSatisfyingExactly(
+                                            equalTo(GEN_AI_SYSTEM, OPENAI),
+                                            equalTo(GEN_AI_OPERATION_NAME, EMBEDDINGS),
+                                            equalTo(GEN_AI_REQUEST_MODEL, MODEL),
+                                            equalTo(GEN_AI_RESPONSE_MODEL, MODEL),
+                                            equalTo(GEN_AI_TOKEN_TYPE, INPUT)))));
+  }
+
+  @Test
+  void connectionError() {
+    OpenAIClient client =
+        wrap(
+            OpenAIOkHttpClient.builder()
+                .baseUrl("http://localhost:9999/v5")
+                .apiKey("testing")
+                .maxRetries(0)
+                .build());
+    OpenAIClientAsync clientAsync =
+        wrap(
+            OpenAIOkHttpClientAsync.builder()
+                .baseUrl("http://localhost:9999/v5")
+                .apiKey("testing")
+                .maxRetries(0)
+                .build());
+
+    EmbeddingCreateParams request =
+        EmbeddingCreateParams.builder()
+            .model(MODEL)
+            .inputOfArrayOfStrings(singletonList(INPUT))
+            .build();
+
+    Throwable thrown = catchThrowable(() -> doEmbeddings(request, client, clientAsync));
+    assertThat(thrown).isInstanceOf(OpenAIIoException.class);
+
+    getTesting()
+        .waitAndAssertTraces(
+            trace ->
+                trace.hasSpansSatisfyingExactly(
+                    maybeWithTransportSpan(
+                        span ->
+                            span.hasName("embeddings text-embedding-3-small")
+                                .hasException(thrown)
+                                .hasAttributesSatisfyingExactly(
```

Comment:
It looks like [testLatestDeps](https://scans.gradle.com/s/e2v5lhvkjn3e6/failures#1) is failing here

```
java.lang.AssertionError: [span [embeddings text-embedding-3-small] attribute keys] 
Expecting actual:
  [gen_ai.operation.name,
    gen_ai.request.encoding_formats,
    gen_ai.request.model,
    gen_ai.system]
to contain exactly in any order:
  [gen_ai.operation.name, gen_ai.request.model, gen_ai.system]
but the following elements were unexpected:
  [gen_ai.request.encoding_formats]
```

You might need to do something like

```suggestion
                                .hasAttributesSatisfyingExactly(
                                    satisfies(
                                        GEN_AI_REQUEST_ENCODING_FORMATS,
                                        val ->
                                            val.satisfiesAnyOf(
                                                v -> assertThat(v).isNull(),
                                                v ->
                                                    assertThat(v)
                                                        .isEqualTo(singletonList("base64")))),
```